labshare-client/src/components/RegistrationSections/SpecificVolunteer.vue

```diff
@@ -1,10 +1,18 @@
 <i18n>
 {
   "en": {
-    "ready": "I am ready to help in the labs"
+    "ready": "I am ready to volunteer! (Can be changed at any point in the profil settings)",
+    "qualification": "Qualification",
```

Comment:
"Qualifications" is usually plural, even if someone has only one, though this might vary by culture and if it would be confusing for someone used to German English it should probably stay like it is here.